docs/api/sql/Function.md

```diff
@@ -474,6 +474,33 @@ Output:
 3.141592653589793
 ```
 
+## ST_BestSRID
+
+Introduction: Returns the estimated most appropriate Spatial Reference Identifier (SRID) for a given geometry, based on its spatial extent and location.  It evaluates the geometry's bounding envelope and selects an SRID that optimally represents the geometry on the Earth's surface. The function prioritizes Universal Transverse Mercator (UTM), Lambert Azimuthal Equal Area (LAEA), or falls back to the Mercator projection.
+
+- For geometries in the Arctic or Antarctic regions, the Lambert Azimuthal Equal Area projection is used.
+- For geometries that fit within a single UTM zone and do not cross the International Date Line, a corresponding UTM SRID is chosen.
+- In cases where none of the above conditions are met, or for geometries that cross the International Date Line, the function defaults to the Mercator projection.
```

Comment:
I've not found how the data line was handled in the implementation. Would you please point out where the date line was handled in `bestSRID` function?

Reply:
It does not handle geometries crossing the IDL appropriately. By the below line I mean - geometries crossing the IDL are mishandled and cause the determined SRID to default to Mercator projection SRID.
The mishandling happens with the `angularWidth` and `angularHeight` calculation; this causes the `xwidth` to be much larger that 6 degrees, which defaults the SRID to mercator.
> In cases where none of the above conditions are met, or for geometries that cross the International Date Line, the function defaults to the Mercator projection.